__test__/Block.spec.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+configure({ adapter: new Adapter() });
+
+import { factory as blockFactory } from '../src/block';
+import { defaultConfig } from '../src/blockConfig';
+
+describe('The Starter Block', () => {
+    const StarterBlock = blockFactory(React);
+
+    describe('when there is no custom data', () => {
+        it('should render the block with the default content', () => {
```

Comment:
Avoid using the term "render" in your test descriptions since you're not using the enzyme `render` method.

Reply:
Let's discuss this offline. Wording aside, I'm not even sure these tests are the best example to provide in the starter.

Reply:
Per offline discussion, snapshot tests will be replaced with a test to show that block renders without errors, similar to the starting test in `create-react-app`.